modular_skyrat/modules/biohazard_blob/code/biohazard_blob_structures.dm

```diff
@@ -125,7 +125,7 @@
 				R.my_atom = src
 				R.add_reagent(/datum/reagent/toxin, 30)
 				var/datum/effect_system/fluid_spread/foam/foam = new
-				foam.set_up(200, location = my_turf, carry = R)
+				foam.set_up(4, location = my_turf, carry = R)
```

Comment:
My only question is, why is this one in a radius of 4 when all of the other ones are in a radius of five?

Reply:
Beats me, it was originally 50 and 40 but I've decided to cut them to a tenth.